client/src/components/RelatedItems/RelatedProducts.jsx

```diff
@@ -6,9 +6,11 @@ import ComparisonModal from './ComparisonModal.jsx';
 function RelatedProducts({currentProductInfo, relatedProductsInfo}) {
   return (
     <div>
-      {relatedProductsInfo.map(relatedProductInfo => {
-        return <ProductCard relatedProductInfo={relatedProductInfo}/>
-      })}
+      <div class="main-gallery js-flickity" data-flickity='{"pageDots": false, "groupCells": "80%"}'>
```

Comment:
Nice find -> and glad you caught the 'className' issue -> sorry I didn't catch it in my earlier review!

Reply:
I think it wasn't showing the class error before because I didn't have the linter installed.